Neos.ContentRepository.Core/Classes/ContentRepository.php

```diff
@@ -66,21 +66,22 @@ final class ContentRepository
      * @internal use the {@see ContentRepositoryFactory::getOrBuild()} to instantiate
      */
     public function __construct(
-        public readonly ContentRepositoryId $id,
-        private readonly CommandBus $commandBus,
-        private readonly EventStoreInterface $eventStore,
-        private readonly EventNormalizer $eventNormalizer,
-        private readonly SubscriptionEngine $subscriptionEngine,
-        private readonly NodeTypeManager $nodeTypeManager,
-        private readonly InterDimensionalVariationGraph $variationGraph,
+        public readonly ContentRepositoryId              $id,
```

Comment:
here is some odd oldscool / wp style formatting in our holy cr :O :O

```suggestion
        public readonly ContentRepositoryId $id,
```